teams.md/docs/csharp/getting-started/running-in-teams.md

```diff
@@ -6,15 +6,15 @@ sidebar_position: 3
 
 Now that your agent is running locally, let's deploy it to Microsoft Teams for testing. This guide will walk you through the process.
 
-## Teams Toolkit
+## Microsoft 365 Agents Toolkit
 
-Teams Toolkit is a powerful tool that simplifies deploying and debugging Teams applications. It automates tasks like managing the Teams app manifest, configuring authentication, provisioning, and deployment. If you'd like to learn what it helps automate, check out [Teams core concepts](/teams/core-concepts).
+Agents Toolkit is a powerful tool that simplifies deploying and debugging Teams applications. It automates tasks like managing the Teams app manifest, configuring authentication, provisioning, and deployment. If you'd like to learn what it helps automate, check out [Teams core concepts](/teams/core-concepts).
```

Comment:
"If you'd like to learn what it helps automate, check out"

"If you'd like to learn more about these concepts, check out"